google-cloud-bigtable/lib/google/cloud/bigtable/value_range.rb

```diff
@@ -65,7 +67,9 @@ def initialize
         ##
         # Sets the row range with the lower bound.
         #
-        # @param value [String] The value. Required.
+        # @param value [String, Integer] The value. Required. If the argument
+        #   is an Integer, it will be encoded as a 64-bit signed big-endian
```

Comment:
Could these changes break existing users by auto converting the value?

Reply:
The `Integer` argument type was not previously declared in the docs or supported.

Reply:
@billyjacobson Does this sound OK to you?

Reply:
Alright that sounds good!

Reply:
@billyjacobson Awesome, can you approve this PR if it all looks good?